src/csharp/Images.cs

```diff
@@ -18,7 +21,7 @@ private Images(IntPtr imagesHandle)
 
         internal IntPtr Handle { get { return _imagesHandle; } }
 
-        public static Images Load(string[] imagePaths)
```

Comment:
This is bothering me. Why do we use a static method `Images.Load()` instead of `new Images(string[] imagePaths)` just like in java:

https://github.com/microsoft/onnxruntime-genai/blob/d12927416fa791802ce73fa795129f1e14f3d42a/src/java/src/main/java/ai/onnxruntime/genai/Images.java#L16